main.css

```diff
@@ -1,507 +1,375 @@
-    from {
-    text-shadow: 0 0 10px #FF8633, 0 0 20px #FF8633, 0 0 30px #FF8633;
- }
- 
- to {
-    text-shadow:  0 0 10px #FF8633,0 0 20px #FF8633, 0 1 30px #FF8633;
- }
-}
+  from {
+    text-shadow: 0 0 10px #ff8633, 0 0 20px #ff8633, 0 0 30px #ff8633;
+  }
 
-.lnumber .number{
-    font-weight: 900;
-    text-shadow: 0px 3px rgba(0,0,0,0.6)
+  to {
+    text-shadow: 0 0 10px #ff8633, 0 0 20px #ff8633, 0 1 30px #ff8633;
+  }
 }
 
-.lnumber .year{
-    font-weight: 700;
-    text-shadow: 0px 3px rgba(0,0,0,0.6)
+.lnumber .number {
+  font-weight: 900;
+  text-shadow: 0px 3px rgba(0, 0, 0, 0.6);
 }
 
-.vehicle .rname{
-    font-weight: 900;
-    text-shadow: 0px 3px rgba(0,0,0,0.6)
+.lnumber .year {
+  font-weight: 700;
+  text-shadow: 0px 3px rgba(0, 0, 0, 0.6);
 }
 
-.launch_site .rtype{
-    font-weight: 900;
-    font-size:15px;
-    font-style:italic;
-    text-shadow: 0px 3px rgba(0,0,0,0.6)
+.vehicle .rname {
+  font-weight: 900;
+  text-shadow: 0px 3px rgba(0, 0, 0, 0.6);
 }
 
-.links a{
-    text-decoration: none;
-    color:inherit;
+.launch_site .rtype {
+  font-weight: 900;
+  font-size: 15px;
+  font-style: italic;
+  text-shadow: 0px 3px rgba(0, 0, 0, 0.6);
 }
 
-.btn{
-    border: 2px solid #FFCA33;
-    background:none;
-    padding:10px 20px;
-    font-size: 20px;
-    margin:10px;
-    /* margin-left: 40px; */
-    color:#FFCA33;
-    transition:0.8s;
-    position: relative;
-    border-radius: 6px;
-    overflow:hidden;
-    cursor: pointer;
-}
-.btn::before{
-    content: '';
-    position: absolute;
-    top: 0; right: 0;
-    width: 0%;
-    height: 100%;
-    background: #ffbb00;
-    transition: .3s linear;
-    z-index: -1;
-}
-.btn:hover:before{
-    width: 100%;
-    left: 0;
-}
-.btn:hover{
-    color: #fff;
-    text-shadow: 1px 1px 1px #f1c14ac7;
-    box-shadow: -1px 2px 7px #f5de69b5;
+.links a {
+  text-decoration: none;
+  color: inherit;
 }
 
-/*Video Modal*/
-.overlay {
-    background-color: rgba(0,0,0,0.6);
-    backdrop-filter: blur(5px);
-    position: fixed;
-    top: 0;
-    left:0;
-    width: 100%;
-    height: 100%;
+.btn {
+  border: 2px solid #ffca33;
+  background: none;
+  padding: 10px 20px;
+  font-size: 20px;
+  margin: 10px;
+  /* margin-left: 40px; */
+  color: #4cc9f0;
+  font-weight: 700;
+  transition: 0.8s;
+  position: relative;
+  border-radius: 30px;
+  overflow: hidden;
+  cursor: pointer;
 }
-
-.overlay__hidden {
-    display: none;
+.btn::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 0%;
+  height: 100%;
+  background: #ffbb00;
+  transition: 0.3s linear;
+  z-index: -1;
 }
-
-.modal {
-    position: fixed;
-    top:18%;
-    left:18%;
-    right:15%;
-    width:fit-content;
-    height:fit-content;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    transition: all .3s ease;
+.btn:hover:before {
+  width: 100%;
+  left: 0;
 }
-
-.modal__hidden {
-    display: none;
+.btn:hover {
+  /* color: #fff; */
```

Comment:
Unnecessary CSS lines can be removed altogether to avoid cluttering the `code base` for other contributors.